<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>526</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="Calculate_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>751</width>
      <height>531</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>801</width>
      <height>601</height>
     </size>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Calculate">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Calculate</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_acos">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>260</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>acos</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_bracketOpen">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>260</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>(</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_sin">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>220</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>sin</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_enter">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>260</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>140</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_0">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>260</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_minus">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>220</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>220</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>220</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ln">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>300</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>ln</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_div">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>140</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>180</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>140</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_asin">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>220</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>asin</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_mult">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>180</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_dot">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>260</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_log">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>300</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>log</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_pow">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>140</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>pow</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_tan">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>180</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>tan</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_cos">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>140</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>cos</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>180</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_atan">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>300</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>atan</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_plus">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>260</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>180</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_mod">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>300</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>mod</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>140</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_sqrt">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>sqrt</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_bracketClose">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>300</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_1">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>220</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>120</y>
        <width>50</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>⌫</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButto">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>400</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_AC">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>300</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>AC</string>
      </property>
     </widget>
     <widget class="QCustomPlot" name="graph" native="true">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>0</y>
        <width>400</width>
        <height>400</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="result_show">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>320</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border-bottom: 1px solid grey;
	background-color: white;
	color: black;
}</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Graphic</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Страница</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
